packages/indexer/src/indexer.ts

```diff
@@ -256,17 +262,18 @@ export async function run<TFilter, TBlock, TRet>(
                   });
 
                   try {
-                    const transformOutput = await indexer.options.transform({
+                    await indexer.options.transform({
                       block,
                       cursor,
                       endCursor,
                       finality,
+                      context,
+                    });
+                    await indexer.hooks.callHook("handler:after", {
+                      block,
+                      finality,
+                      endCursor,
```

Comment:
I think this is what we were discussing yesterday. Notice that this handler is _inside_ the transaction so there is still a chance that the handler is called and the transaction fails (e.g. because of a connection error to the db). If we used this one for persistence and that happens there would be a data loss.

We should add a similar hook (just pass `finality` and `endCursor`) called `transaction:commit` _after_ the `transaction(...)` callback.